.github/workflows/test.yml

```diff
@@ -110,12 +110,16 @@ jobs:
     permissions:
       contents: read
     steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
       - name: Build and push Docker image
         uses: docker/build-push-action@v4
         with:
+          platforms: linux/amd64,linux/arm64
```

Comment:
would it make sense to do multi-platform builds only for release? 9 minutes for a blocking test on a PR sounds above the acceptable threshold

Reply:
That depends on how much you want to support arm. The arm container build can fail when updating requirements.

Reply:
true, we should probably either support it 100% or not. Currently the 9 minutes build time seems to be a blocker - do you have any idea how we could speed it up? GitHub doesn't seem to provide hosted ARM workers at the time

Reply:
You can also provide self hosted runners for arm, which is what we do as an org, but that might not be feasible for an open source project.

Reply:
sounds like a fair option, feel free to update the Docker image to use Debian and I'll be happy to merge if the build time is reasonable (say <2-3 minutes)